README.md

```diff
@@ -129,7 +128,7 @@ In this section, we'll use the [Azure CLI](https://docs.microsoft.com/cli/azure/
 
    ```bash
    webAppName=my-keystone-app
-   az webapp create --resource-group $rgName --name $webAppName --plan $appPlanName --runtime "node|10.14"
+   az webapp create --resource-group $rgName --name $webAppName --plan $appPlanName --runtime "NODE|14-lts"
```

Comment:
This was installing Node 10.x, got the correct version by running this:

<img width="408" alt="NODE112-lts," src="https://user-images.githubusercontent.com/737821/138193118-e56dfd10-44cb-41c6-a49f-a04626821c60.png">

Reply:
This installed v14.15.1

<img width="631" alt="login Thu Oct 21 005540 2021 from 172 16 0 2" src="https://user-images.githubusercontent.com/737821/138193148-e7988ba1-deea-4ea8-acf9-c7e11eed7c98.png">